### PR TITLE
PR for adding ReviewList test

### PR DESCRIPTION
**Pull request by aimeekang**
PR for adding ReviewList test

### CODE REVIEW

File: client/source/components/RR/tests/ReviewList.test.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen, cleanup } from '@testing-library/react';
+import { reviews } from './RRdata.js';
```
Comment (AedonGrunGott) [inline client/source/components/RR/tests/ReviewList.test.jsx]: You are destructuring here, but export a single default from `RRData`

Comment (AedonGrunGott) [reply client/source/components/RR/tests/ReviewList.test.jsx]: Also, this test currently does not pass.

File: client/source/App.jsx [hunk 2]
```diff
@@ -28,4 +28,4 @@ const App = function CreateAppComponent() {
   );
 };
 
-export default App;
+export default App;
```
Comment (AedonGrunGott) [inline client/source/App.jsx]: EOF needs a new line. You probably just editing this file by mistake.

File: client/source/components/RR/tests/RRdata.js [hunk 3]
```diff
@@ -0,0 +1,674 @@
...
+        {
+          id: 1236197,
+          url: 'https://images.unsplash.com/photo-1534011546717-407bced4d25c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2734&q=80'
+        }
+      ]
+    }
+  ]
+};
+
+export default reviews;
```
Comment (AedonGrunGott) [inline client/source/components/RR/tests/RRdata.js:92]: Do you need a 674 line long block of test data? What about just 3 or 4 reviews to switch between?

File: client/source/components/RR/styles/RR.styled.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  justify-content: space-around;
+  align-items: center;
+  border: 0.1em solid black;
+`;
+
+export const ReviewListContainer = styled.div`
+
+`;
+
+// export const ReviewTileContainer = styled.div`
```
Comment (AedonGrunGott) [inline client/source/components/RR/styles/RR.styled.js]: No need to leave development-only comments in a merge to the production branch.

File: client/source/index.jsx [hunk 5]
```diff
@@ -9,5 +9,5 @@ import AppHolder from './AppHolder';
 // MAIN
 const root = createRoot(document.getElementById('root'));
 root.render(
-  <AppHolder />,
+  <AppHolder />
```
Comment (AedonGrunGott) [inline client/source/index.jsx:12]: **_DIE COMMA!_**

**Review by AedonGrunGott — CHANGES_REQUESTED** [review]
A few minor notes, but because the test doesn't pass (and has a few errors) I'm withholding parental approval for now.

File: client/source/index.jsx (hunk 5 above)
Comment (aimeekang) [reply client/source/index.jsx:12]: the comma-dangle must go

File: client/source/components/RR/tests/ReviewList.test.jsx (hunk 1 above)
Comment (aimeekang) [reply client/source/components/RR/tests/ReviewList.test.jsx]: Updated the method with .toContain and added jest environment up top

File: client/source/components/RR/tests/RRdata.js (hunk 3 above)
Comment (aimeekang) [reply client/source/components/RR/tests/RRdata.js:92]: Updated to 4 reviews

File: client/source/App.jsx (hunk 2 above)
Comment (aimeekang) [reply client/source/App.jsx]: Updated

File: client/source/components/RR/styles/RR.styled.js (hunk 4 above)
Comment (aimeekang) [reply client/source/components/RR/styles/RR.styled.js]: Updated

**Comment by aimeekang** [comment]
@AedonGrunGott I have made the changes requested. Please take a look in the morning!

**Review by avinashi10 — APPROVED** [review]
Cross referenced with August's comments and your follow-up comments, everything there looks great!

**Review by BrettEastman — APPROVED** [review]
Looks good!

**Review by AedonGrunGott — APPROVED** [review]

**Comment by AedonGrunGott** [comment]
button 0 0

**Pull request merged**